cabot_alert_hipchat/models.py

```diff
@@ -45,21 +96,8 @@ def send_alert(self, service, users, duty_officers):
         })
         message = Template(hipchat_template).render(c)
         self._send_hipchat_alert(message, color=color, sender='Cabot/%s' % service.name)
+        self._send_images_to_hipchat(service)
```

Comment:
hmm yeah so we can add the message to an image and post both at once. We can't post multiple images at once though (which I'm doing if checks are failing). I could post the message with either the first or last message though?